### ISSUE TITLE
WCAG 2 readiness

### ISSUE THREAD

**Issue by fkolar** [SAP/fundamental#1269]
When we are discussing and opening question about FIORI 3 , It is also  required by Fiori 3 bronze level to be able to meet accessibility WCAG Conformance Level A.​

https://webaim.org/standards/wcag/checklist

I understand some of the things from the check list cannot be applied to this project , but are you aware and are you monitoring your progress to go side by side with WCAG?

Frank

**Comment by saad-mo** [comment]
Hi @fkolar 

An audit has been performed on all fundamentals components and we will work on making applicable updates. We will soon be adding issues to our backlog and release according to priority. 

Thanks.
